ui/server/src/main/scala/pl/touk/nussknacker/ui/NussknackerApp.scala

```diff
@@ -116,7 +116,8 @@ object NussknackerApp extends App with Directives with LazyLogging {
 
     Initialization.init(modelData.mapValues(_.migrations), db, environment, config.getAs[Map[String, String]]("customProcesses"))
 
-    val managementActor = ManagementActor(environment, managers, processRepository, deploymentProcessRepository, subprocessResolver)
+    val managementActor = system.actorOf(
```

Comment:
is it needed now?

Reply:
It is a good practise to have a `Props` and create actors in one place in application.